src/System.Management.Automation/engine/Modules/ModuleSpecification.cs

```diff
@@ -165,6 +165,35 @@ internal static Exception ModuleSpecificationInitHelper(ModuleSpecification modu
             return null;
         }
 
+        internal string GetRequiredModuleNotFoundVersionMessage()
+        {
+            if (RequiredVersion != null)
+            {
+                return StringUtil.Format(Modules.RequiredModuleNotFoundRequiredVersion, Name, RequiredVersion);
+            }
+            else
```

Comment:
We could remove the else and simplify cascading. Below too. This could make the code more readable.